src/tpm2_wrap.c

```diff
@@ -71,3 +71,337 @@ int wolfTPM_UnloadHandle(word32* handle)
 
     return rc;
 }
+
+int wolfTPM2_NVReadPublic(word32 nvIndex)
+{
+    int rc = TPM_RC_SUCCESS;
+    NV_ReadPublic_In in;
+    NV_ReadPublic_Out out;
+
+    in.nvIndex = nvIndex;
+    rc = TPM2_NV_ReadPublic(&in, &out);
+    if (rc != TPM_RC_SUCCESS) {
+        printf("TPM2_NV_ReadPublic failed %d: %s\n", rc, wolfTPM2_GetRCString(rc));
```

Comment:
unwrapped printf statement could cause portability issues. Maybe surround it with the DEBUG_WOLFTPM macro.